model/src/main/java/de/danoeh/antennapod/model/feed/FeedItem.java

```diff
@@ -427,6 +432,14 @@ public void removeTag(String tag) {
         tags.remove(tag);
     }
 
+    public String getChapterUrl() {
```

Comment:
Everywhere else you call it `podcastIndexChapterUrl`. I think it would be good to mention it here, too. Or does that make the method name too long?